nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLBinariesIndexingOverride.java

```diff
@@ -186,4 +186,28 @@ public void testExcludeFieldBlob() throws Exception {
         assertEquals(0, res.size());
     }
 
+    @Test
+    public void testEnforceFulltextFieldSizeLimit() throws Exception {
+        DocumentModelList res;
+        for (int i = 0; i < 2; i++) {
+            String namePrefix = i == 0 ? "reg" : "big";
```

Comment:
I'd do `boolean regular = i == 0` and use this instead (in two places) for clarity.